lib/contracts/contributor.js

```diff
@@ -8,9 +9,13 @@ class Contributor extends Record {
 
   getById(id) {
     return this.functions.getContributorById(id)
-      .then((data) => {
+      .then(data => {
         return this.ipfs.catAndMerge(data, ContributorSerializer.deserialize);
-      });
+      })
+      .then(data => {
+        data.balanceInt = KreditsFormatter(data.balance);
```

Comment:
What's the benefit of doing this here instead of before the return in the first `this` above?

Reply:
`catAndMerge` returns a promise, so I only can do it in a next `then`, no?

Reply:
I thought `getContributorById` already includes the balance. In which case it doesn't matter what's returned afterwards, as it would already include the formatted balance beforehand.

Reply:
ah, true. before passing it to IPFS. :+1: